feat: add url_encode and url_decode std lib functions (#4287)
grafana/alloy: docs/sources/reference/components/prometheus/prometheus.exporter.postgres.md

}
```

### Escape special characters in postgres url

> If your PostgreSQL connection string includes special characters for e.g. password (`@`, `:`, `/`, etc.), you should wrap the password using `encoding.url_encode`.

Reviewer: ```suggestion
If your PostgreSQL connection string includes special characters, for example `@`, `:`, or `/`, you must use `encoding.url_encode` to wrap the password .
```
